Makefile

```diff
@@ -128,7 +128,8 @@ help: ## help on rule's targets
 
 
 test_python_version: ## Check Python version, throw error if compilation would fail with the installed version
-	python ./scripts/test_python_version.py
+	# Checking python version
+	@.venv/bin/python ./scripts/test_python_version.py
```

Comment:
I changed this in the other PR to run with `uv run`

Reply:
ok I will revert then mine. I just saw this was failing

Reply:
ok so as discussed with @pcrespov this is actually correct, as we want to test the python version **inside** the venv